### PR TITLE
1084 qiita db

### PR DESCRIPTION
**Pull request by josenavas**
Built on top of #1204 

Makes qiita_db to pass al tests. Still failures expected on qiita-ware and qiita-pet.

### CODE REVIEW

File: qiita_db/commands.py [hunk 1]
```diff
@@ -153,7 +153,7 @@ def load_sample_template_from_cmd(sample_temp_path, study_id):
     return SampleTemplate.create(sample_temp, Study(study_id))
 
 
-def load_prep_template_from_cmd(prep_temp_path, raw_data_id, study_id,
+def load_prep_template_from_cmd(prep_temp_path, study_id,
                                 data_type):
```
Comment (antgonza) [inline qiita_db/commands.py]: you can bring this line up, no?

**Comment by antgonza** [comment]
1 minor comment :+1:

**Comment by josenavas** [comment]
Done!

File: qiita_db/test/test_study.py [hunk 2]
```diff
@@ -586,26 +585,20 @@ def test_retrieve_data_types_none(self):
         self.assertEqual(new.data_types, [])
 
     def test_retrieve_raw_data(self):
-        self.assertEqual(self.study.raw_data(), [1, 2, 3, 4])
+        self.assertEqual(self.study.raw_data(), [1])
```
Comment (ElDeveloper) [inline qiita_db/test/test_study.py]: There used to be four raw datas associated to this study but now there's only one? How would that happen?

**Comment by ElDeveloper** [comment]
1 question, but :+1: (for the most part :100: ).

File: qiita_db/test/test_study.py (hunk 2 above)
Comment (josenavas) [inline qiita_db/test/test_study.py]: I removed them in populate_test_db.sql. Keeping them would require adding 3 more prep templates there, which is a lot of work...

Comment (ElDeveloper) [inline qiita_db/test/test_study.py]: Thanks!

**Pull request merged**